Products/ZenUI3/browser/resources/js/zenoss/devicemanagement/Administration.js

```diff
@@ -11,6 +11,8 @@
 
 Ext.ns('Zenoss', 'Zenoss.devicemanagement');
 
+    timeZone = "";
```

Comment:
1. This should be declared with `var` so that it's only visible within this closure.
2. `timeZone` is *not* actually used anywhere that I can see.